spec/controllers/default_bills_controller_spec.rb

```diff
@@ -0,0 +1,247 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DefaultBillsController, type: :request do
+  describe '#default_bills_controller' do
+    let!(:current_user) { create(:user) }
+    let!(:another_user) { create(:user, name: 'Second user', slug: 'second-user-1', email: 'second.user@mail.com') }
+
+    let!(:current_default_bill) { create(:default_bill, user_id: current_user.id) }
+    let(:second_default_bill) do
+      create(:default_bill, name: 'Second default bill', slug: 'second-default_bill-2', user_id: another_user.id)
+    end
```

Comment:
Isso aqui vc não tá usando em lugar nenhum, acho que vc poderia usar `let!` (com `!`) e daí vc arruma o primeiro teste do `#index` que vai começar a retornar 2 resultados ao invés de 1

Reply:
Na verdade a intenção é essa, já que essa segunda conta é de outro usuário, e no #index eu puxo as contas do `current_user`, e checo se essa conta não aparece lá

Reply:
Entendi, mas mesmo assim, essa conta `second_default_bill` nunca é criada no banco de dados.

Vc precisa trocar para:
```suggestion
    let!(:second_default_bill) do
      create(:default_bill, name: 'Second default bill', slug: 'second-default_bill-2', user_id: another_user.id)
    end
```

Pq daí sim essa default_bill vai ser criada no banco de dados e o seu teste vai fazer sentido